receiver/awsxrayreceiver/config.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsxrayreceiver
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/config/configtls"
+)
+
+const (
+	typeStr = "aws_xray"
+	version = "1.0.0"
+)
+
+// Config defines the configurations for an AWS X-Ray receiver.
+type Config struct {
+	// The Endpoint field in ReceiverSettings represents the UDP address
+	// and port on which this receiver listens for X-Ray segment documents
+	// emitted by the X-Ray SDK.
+	configmodels.ReceiverSettings `mapstructure:",squash"`
+	// squash ensures fields are correctly decoded in embedded struct
+	// https://godoc.org/github.com/mitchellh/mapstructure#hdr-Embedded_Structs_and_Squashing
```

Comment:
That comment is probably overkill, given how `squash` is used throughout this repo. Also I think its fairly well implied from the name?

Reply:
Thanks James. I will remove this in the followup PR I'm working on now.

Reply:
@james-bebbington Seems like the PR is still blocked on Tigran's approval. Could you remove him from the list of reviewers. I tried but couldn't do it myself. Thanks!